blog/_posts/2018-11-18-TeamSectumsempraGHCI.md

```diff
@@ -0,0 +1,57 @@
+After finally gathering in the main hall, all of us eagerly waited for the keynotes to begin. As the welcome presentation by Geetha Kanan, the organizer of the conference commenced, all of us cheered and hooted with excitement while she walked us through the beautiful journey 
+of GHCI through the years, as well as the women behind it all. We all were so thrilled to see an all  women team behind such a reputed and well-appraised conference. 
+Soon after, the first keynote by Lori Beer took place, and she gave us such important lessons while undertaking our journey in technology. 
+She emphasized how its so essential to dream big. With her talk, we all envisioned to carry forward the dream of achieving the  “Fifty Fifty ratio in tech by 2025”.i As I write this down, I reminisce her words, which ring repeatedly loud and clear, in my head: “Be bold, be
+brave, be yourself, be unstoppable.” 
+One of the talks that stood out, and for me, it definitely was the highlight of the day was by Vaishali Kasture,one of the keynote speakers. 
+She educated us on the importance of the art of selling, and how crucial it is to be able to put yourself out there, to be able to derive the best for yourself, 
+your company as well as your clientele. 
+Citing her own example, she mentioned proudly and just as casually , that she came  from a non-tech background herself but was in the tech business for quite long now, 
+and to top it all, she was addressing a gathering of 5000 female engineers, which was quite unconventional, thereby highlighting the essential need for all of us to learn 
+how to negotiate well, and to sell as well. While talking about unconscious bias, she pointed out how it is common for us it is to assume that selling and negotiating are associated with 
+masculinity its time to eliminate that. Moreover, she engaged the audience through interesting anecdotes from her experience so far. An interesting thought she provoked, stuck with me. “Out west they say the squeaky wheel gets the grease, but here we say that the nail that sticks out gets the hammer.” And that was one of the key takeaways for me, that is, mastering the art of ethics, as well as negotiation, and doing it with a lot of empathy and ethics. All the speakers shared their experiences from their tech journeys, and 
+left us with thought provoking words and all the more encouragement to be engaged with tech as a career, and do things right. Their words, energy, conviction and most importantly, the motivation with which they spoke, spread infectiously and so quickly among the crowds and while young girls quickly soaked their minds with all that was pouring in, 
+I too, not falling behind gathered it all to take back home with me, not only to cherish, but to carry it forward in my journey in tech. 
+As the day concluded, with happy hearts and craving minds, and we looked forward to know what’s in store for the upcoming days. 
+
+![Alt Text](/img/blog/2018/teamSectumsempra-GHCI2018.jpg)
```

Comment:
Ror this image, please use the tips I suggested in the last PR https://github.com/rails-girls-summer-of-code/summer-of-code/pull/577: 
- using an alt text that is descriptive
- changing the source to only “Avneet Kaur”
- making the image smaller as suggested [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images)